### PR TITLE
added armor peircing levels to medic attacks

### PR DESCRIPTION
**Pull request by bleehu**
To use new systems.

### CODE REVIEW

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 1]
```diff
@@ -39,17 +39,17 @@ Adrenaline Cost:30
...
 /////	Fast Spike
 Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	1 Action
 Range:		40m
-Effect:		16 % chance to miss. Deal 52 piercing damage.
-If target has no shield left and is still alive, apply a 
+Effect:		16 % chance to miss. Deal 52 piercing damage. AP level 2.
+If the needle defeats the target's armor and the target is still alive, apply a 
 poison payload of your choice.
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: question of mechanics, shouldn't the payload by applied before the needle is launched?

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 2]
```diff
@@ -39,17 +39,17 @@ Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	2 Actions
 Range:		40m
-Effect:		10 % chance to miss. Deal 52 piercing damage.
-If target has no shield left and is still alive, apply a 
+Effect:		10 % chance to miss. Deal 52 piercing damage. AP level 3.
+If the needle penetrates the targets armor and is still alive, apply a
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: I don't think piercing damage is anywhere in the basic rules... is it defined somewhere else? Or is 'piercing' flavor text?

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 3]
```diff
@@ -59,16 +59,17 @@ Duration: 	2 Actions
 Range:		30m
 Effect:		You launch off three harmasist spikes in quick succession using both
 hands. Each has an 18 % chance to miss. You may pick up to three targets to 
-launch at. Each spike that hits deals 52 piercing damage. If target has no 
-shield left and is still alive, apply a poison payload of your choice.
+launch at. Each spike that hits deals 52 piercing damage and has an armor piercing 
+level of 3. If the needles defeat the armror of the target and the target is still alive, 
+apply a poison payload of your choice.
 
 /////	Peircing Spike
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: Piercing*

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 2 above)
Comment (1sourcecontrol) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: Also, I don't see 'Harmasist Spike' in the malpractice tree image that you loaded a while ago. (Also, you misspelled 'Piercing' in the malpractice tree image).

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 4]
```diff
@@ -248,7 +249,8 @@ Duration: 	almost instantaneous
 Range:		Payload (use in combination with other powers)
 Effect:		A biologic target aflicted with this poison takes 180 damage that 
 ignores shields, silences the target and prevents them from casting nanite
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:252]: Only plasma and laser things get blocked by shields. (unless you are using a plasma shield [I think it's called], like the paladin). So... unless you mean that it ignores _all_ forms of shields, this sort of text is unnecessary.

Also, I don't think silencing is defined anywhere. Do you mean silencing in the mouth-speaking form? Because I first thought you meant silence in the league-of-legends sense, where you can't cast spells.

**Review by 1sourcecontrol — COMMENTED** [review]
Lots of things not related to armor piercing level, but I've never looked in depth at medic before.

Don't forget to make a pass through this and convert the miss chance system to d10 scale if you are using these like weapon attacks with perception combat mod. I think consistency across classes would be beneficial. 

Keep in mind, player level 5 is when armor levels 3 & 4 become available. By player level 15, I'd expect quite a few enemies with armor level around 5-8, which makes abilities with static AP levels harder to use. Especially since you can't just throw an ability out and get a better one like you would a normal rifle. 
If you have some way to upgrade existing medic abilities, or higher level needles get better AP levels, that would help a lot.

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 1 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: Applied to target, not to needle.

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 2 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: I need to update the image, but the machine that I develop on doesn't handle images well. I'll update that in a non-Combat 2.0 branch, because it needs to be updated regardless of the status of Combat 2.0

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 4 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:252]: It's supposed to be in the League of Legend sense. I'll clarify that.

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 5]
```diff
@@ -59,16 +59,17 @@ Duration: 	2 Actions
...
 
 /////	Peircing Spike
 Adrenaline Cost:35
 Needle Cost: 1
 Duration: 	2 Actions
 Range:		45m
-Effect:		10 % chance to miss. Deal 42 piercing damage.
-ignores sheilds and balistic armor. Apply a poison payload of your choice.
+Effect:		10 % chance to miss. Deal 42 piercing damage. AP level 4.
+if the  needle defeats the target's armor, apply a poison payload of your choice.
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: Now that I understand the purchase hierarchy for medic abilities (thanks for updating your image :) , is it your intention that Harmasist, Fast, and Piercing Spike should be roughly equal in power? Or should the abilities be more powerful higher up the tree they go?
Right now, it looks like they are all more-or-less equal.

If you want them to be better the further up the tree you go, my suggestion is to (as a minimum) increase piercing spike's AP level to 5 and Fast spike's adrenaline cost by 5 and AP level to 3.  

Or decrease Harmasist to ap lvl 2, increase fast spikes adrenaline cost by 5, and leave piercing spike as it is. (If Harmasist is considered a lvl1 ability, the normal AP level range is a 0 to 2, (unless you are using a long rifle).)

Comment (bleehu) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: wilco! :)

**Comment by 1sourcecontrol** [comment]
Are the miss chances supposed to be different than the d10 combat system?

**Comment by bleehu** [comment]
<facepalm> Nope.

**Comment by bleehu** [comment]
Updated

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 6]
```diff
@@ -30,7 +30,7 @@ Duration: 	Free Action
 Range:		Self
 Effect:		Increase move speed to 12m per action
 and get a +30 to accrobatics checks. Effect persists for 1 turn. While 
-in a medical emergency, the Doc suffers a +28% chance to miss, 'cause
+in a medical emergency, the Doc suffers a +2.5 chance to miss, 'cause
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:33]: I've been keeping the miss chances to a whole number value unless it's an attachment. If you want to use a decimal number you'll need to define the rules for rounding that number (rounding up/down/w.e. after accounting for all other factors).
The pickle I've been dealing with is, "when do we make the player round the numbers?" To make it simple, I've designed it thus far that the player rounds (drops the remainder) miss chance values for weapons when they go to write the values down in their 3x3 table.
So in summary, all effects in the middle of play have to either define how they are supposed to be rounded, or they should be whole number values.

Skill checks are a separate beast, of course, since they aren't d10 scale.

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 7]
```diff
@@ -39,36 +39,38 @@ Adrenaline Cost:30
...
 /////	Fast Spike
-Adrenaline Cost:30
+Adrenaline Cost:35
 Needle Cost:	1
 Duration: 	1 Action
 Range:		40m
-Effect:		16 % chance to miss. Deal 52 piercing damage.
-If target has no shield left and is still alive, apply a 
-poison payload of your choice.
+Effect:		2 chance to miss. Deal 52 piercing damage. AP level 3.
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:51]: soooooooo, is "Piercing Damage" flavor text or does it need to be defined?

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
Left 2 comments, but the comments affect the rest of the changes.

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 6 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:33]: Can't perception mod be a half number? Or am I miss-remembering that?

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 7 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:51]: Flavor, really. Think I should yank it?

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 6 above)
Comment (1sourcecontrol) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:33]: It can be a half number. But what happens when it's a whole number, such that your miss chance leaves a half number?

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 7 above)
Comment (1sourcecontrol) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:51]: I think yanking it is a good idea. I don't mind it now that I know it's flavor text, but nobody else knows, and several games I've played use 'piercing' as a damage type.

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 6 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:33]: Then you already wrote the rule that it rounds down? I think dropping it to 2.0 would be too weak, and 3.0 would be too strong.

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 7 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:51]: wilco

**Comment by bleehu** [comment]
Fixed Spelling in PR #155

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 6 above)
Comment (1sourcecontrol) [reply play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:33]: Oops, just saw this. The rounding rules apply before abilities come in, I don't have any abilities that use halves... (I hope?)

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
Per our conversation, need to round miss chances.
If you feel that you can't balance a "spell/attack" with the rounding, I can totally give suggestions on altering the attacks!

**Comment by 1sourcecontrol** [comment]
@bleehu  You're merging into the C2.0 superbranch. Can you switch the merge to dev so we can delete the superbranch?

**Comment by 1sourcecontrol** [comment]
@bleehu or if you want to, you can merge into the superbranch then PR c2.0 superbranch and merge it.

**Comment by bleehu** [comment]
PR canceled due to merge of super branch. Changes will be made and a new PR will be made against the dev branch.
